#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_path_changes.dpatch by Jose Carlos Garcia Sogo <jsogo@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Debian-specific pathnames.

@DPATCH@
diff -urNad smokeping-2.2.4~/etc/basepage.html.dist smokeping-2.2.4/etc/basepage.html.dist
--- smokeping-2.2.4~/etc/basepage.html.dist	2007-09-17 11:04:25.000000000 +0300
+++ smokeping-2.2.4/etc/basepage.html.dist	2007-09-17 11:04:29.000000000 +0300
@@ -43,10 +43,10 @@
 -->
 </STYLE>
 
-<script src="cropper/lib/prototype.js" type="text/javascript"></script>      
-<script src="cropper/lib/scriptaculous.js?load=builder,dragdrop" type="text/javascript"></script>
-<script src="cropper/cropper.js" type="text/javascript"></script>
-<script src="cropper/smokeping-zoom.js" type="text/javascript"></script>
+<script src="/smokeping/javascript/prototype/prototype.js" type="text/javascript"></script>      
+<script src="/smokeping/javascript/scriptaculous/scriptaculous.js?load=builder,effects,dragdrop" type="text/javascript"></script>
+<script src="/smokeping/javascript/cropper/cropper.js" type="text/javascript"></script>
+<script src="/smokeping/javascript/smokeping-zoom.js" type="text/javascript"></script>
 
 
 
